tests/test_sort_package_xml.py

```diff
@@ -14,12 +13,12 @@
         "no-format",
     ],
 )
-def test(case: str, datadir: Path):
+def test(case: str, datadir: Path, tmp_path: Path):
     input_file = datadir / f"{case}.input.xml"
     answer_file = datadir / f"{case}.answer.xml"
 
     # Create a temporary file to prevent overwriting original files
-    tmp_file = Path(NamedTemporaryFile(mode="w", suffix=".xml").name)
+    tmp_file = tmp_path / f"{case}.input.xml"
```

Comment:
Where is `tmp_path` from?

Reply:
tmp_path is https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
I see, looks nice. :+1: